website/docs/r/ip_space_ip_allocation.html.markdown

```diff
@@ -105,8 +126,10 @@ The following arguments are supported:
   * `IP_PREFIX` - allocates subnets. **Note** field `prefix_length` is required to allocate IP
     Prefix
 * `prefix_length` (Optional) Required when `type=IP_PREFIX`
-* `value` - (Optional; VCD *10.4.2+*) An option to request a specific IP or subnet from IP Space
-* `usage_state` - (Optiona) Not required unless manual IP reservation is required which can be
+* `value` - (Optional; VCD *10.4.2+*) An option to request a specific IP or subnet from IP Space.
+  **Note:** This field does not support IP ranges because it would cause multiple allocations
+  created in one resource. Please use multiple resource instances to allocate IP ranges.
+* `usage_state` - (Optional) Not required unless manual IP reservation is required which can be
   enabled `USED_MANUAL`. Value `UNUSED` must be set to release manual allocation of IP.
```

Comment:
Though it's not from this PR, I just noticed that this sentence is funky. Would be good to fix it. Something between the lines (this is just an example to make a point, I'm not sure it's correct):

```suggestion
* `usage_state` - (Optional) Value  `USED_MANUAL` enables manual IP reservation. Value `UNUSED` must be set to release manual allocation of IP.
```

Reply:
Indeed it could be clearer. What if I prepend such a sentence to your proposal as I don't want to confuse users. 
> Only required for manual reservations.

In the end it could be:

* `usage_state` - (Optional) Only required for manual reservations. Value `USED_MANUAL` enables manual IP reservation. Value `UNUSED` must be set to release manual allocation of IP.

Reply:
Working on the above further:

* `usage_state` - (Optional) Only used with manual reservations. Value `USED_MANUAL` enables manual IP reservation. Value `UNUSED` is set to release manual allocation of IP.

Reply:
I like this last suggestion - applied.